src/ontology/cl-edit.owl

```diff
@@ -28103,6 +28106,37 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24855283"^^xsd:string) r
 AnnotationAssertion(rdfs:label obo:CL_4030017 "epithelial cell of late distal convoluted tubule")
 EquivalentClasses(obo:CL_4030017 ObjectIntersectionOf(obo:CL_0000066 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005102)))
 
+# Class: obo:CL_4030019 (kidney connecting tubule intercalated cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:9890303"^^xsd:string) obo:IAO_0000115 obo:CL_4030019 "Epithelial cell that is part of the connecting tubule with important roles in the regulation of acid-base homeostasis."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030019 <https://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030019 "2022-06-29T11:25:22Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:CL_4030019 "kidney connecting tubule intercalated cell")
+EquivalentClasses(obo:CL_4030019 ObjectIntersectionOf(obo:CL_0005010 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005097)))
+SubClassOf(obo:CL_4030019 obo:CL_1000768)
+
+# Class: obo:CL_4030020 (kidney connecting tubule intercalated cell type A)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:2254448"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:9890303"^^xsd:string) obo:IAO_0000115 obo:CL_4030020 "Epithelial cell that is part of the connecting tubule and secretes protons via an apical H+-ATPase and H+/K+ exchanger and reabsorbs bicarbonate by a Cl-/HCO3- exchanger (AE1) located in the basolateral plasma membrane."^^xsd:string)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> obo:CL_4030020 <https://orcid.org/0000-0002-1773-2692>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4030020 "2022-06-29T12:48:43Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24937428"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4030020 "connecting tubule A-IC"^^xsd:string)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:24937428"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4030020 "connecting tubule alpha-intercalated cell"^^xsd:string)
+AnnotationAssertion(rdfs:label obo:CL_4030020 "kidney connecting tubule intercalated cell type A")
+SubClassOf(obo:CL_4030020 obo:CL_0005011)
```

Comment:
I don't understand this subclass axiom - it needs to be mentioned in the def.

Reply:
@shawntanzk, thanks for pointing that out. It made me realise that I could drop the subclasses and add the EA:

`'renal alpha-intercalated cell' and 'part of' some 'renal connecting tubule'`

Same for type B / beta cell.

Does that make sense?

Reply:
I tested this EA and saw some unexpected results. I think it is due to axioms that are on 'renal alpha-intercalated cell':

<img width="371" alt="Screenshot 2022-06-30 at 14 50 11" src="https://user-images.githubusercontent.com/95643797/176694739-60ce0fa4-f475-42fb-8812-074257c88b48.png">

These 'part of' axioms refer to distinct regions of the kidney and more than one cannot be true at the same time, i.e., a cell cannot be part of the collecting duct and the connecting tubule at the same time. Should these be removed or concatenated with 'or'? 
Note that after synchronising, it is inferred that a 'kidney connecting tubule intercalated cell type A' is equivalent to a 'renal alpha-intercalated cell', when it is really a subclass.

Reply:
Removed these 'part of' axioms and did some label renaming for consistency.

Reply:
I don't think theres is 'or' reason with the reasoners we use? but anyway what we usually do is find a term that is broad enough to cover it - so maybe kidney?